docs/docusaurus/docs/guides/expectations/creating_custom_expectations/add_custom_parameters.md

```diff
@@ -0,0 +1,91 @@
+   "column_B",
+   "mostly",
+)
+```
+The tuple in the example includes the two columns being evaluated and the `mostly` parameter. These parameters are passed as a part of your ExpectationConfiguration and are available to the Expectation `_validate` method.
+
+## Use case
+
+In this use case, a `condition_value_keys` tuple and a `condition_domain_keys` tuple are added to ColumnPairMapMetrics, MulticolumnMapMetrics, and ColumnMapMetrics. The `condition_value_keys` tuple supplies the arguments needed to compute your Metric, and the `condition_domain_keys` tuple defines the domain on which the Metric operates.
+
+This is how the tuples appear in the MulticolumnMapMetrics Metric:
+
+```python
+condition_domain_keys = (
+   "batch_id",       
+   "table",       
+   "column_list",       
+   "row_condition",       
+   "condition_parser",       
+   "ignore_row_if",   
+)   
+condition_value_keys = ("sum_total",)
+
+ColumnAggregateMetrics, TableMetrics, and QueryMetrics can similarly define a value_keys tuple or a domain_keys tuple:
+
+value_keys = ("column",)
+domain_keys = ("query",)
+```
+
+After the attributes are added to the Expectation and the Metric, the custom parameters can be passed into and used within the individual Metric functions. For example, this is how it appears in the ColumnValuesBetween Metric:
+
+```python
+classColumnValuesBetween(ColumnMapMetricProvider):   
+condition_metric_name = "column_values.between"   
+condition_value_keys = (       
+   "min_value",       
+   "max_value",       
+   "strict_min",       
+   "strict_max",       
+   "parse_strings_as_datetimes",
+   "allow_cross_type_comparisons",
+)
+@column_condition_partial(engine=PandasExecutionEngine)   
+def_pandas(       
+   cls,       
+   column,       
+   min_value=None,       
+   max_value=None,       
+   strict_min=None,       
+   strict_max=None,       
+   parse_strings_as_datetimes: bool = False,
+   allow_cross_type_comparisons=None,       
+   **kwargs   
+):
+```
+To view the full code that you would use to pass custom parameters in a Custom Expectation, see [expect_column_values_to_be_lat_lon_coordinates_in_range_of_given_point.py](https://github.com/great-expectations/great_expectations/blob/develop/contrib/great_expectations_geospatial_expectations/great_expectations_geospatial_expectations/expectations/expect_column_values_to_be_lat_lon_coordinates_in_range_of_given_point.py).
+
+## kwarg-based access
+
+GX recommends using custom parameters to create validations and optimize workflows. However, you can also use `kwarg` syntax to define Metrics for your Expectations. An example `kwarg` implementation is provided in [expect_column_values_to_be_lat_lon_coordinates_in_range_of_given_point.py](https://github.com/great-expectations/great_expectations/blob/develop/contrib/great_expectations_geospatial_expectations/great_expectations_geospatial_expectations/expectations/expect_column_values_to_be_lat_lon_coordinates_in_range_of_given_point.py).
```

Comment:
While this totally works, it's important to note that all these kwargs still need to be set in a value_keys tuple before being accessible through kwargs.

Reply:
@austiezr I have made an attempt to add the information you requested. Please review.